app/models/post.rb

```diff
@@ -2,7 +2,6 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
```

Comment:
- [ ] It would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private.

```ruby
 after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```